File: src/codes/Users_Manual/modulation/fourier_3.cpp
//~#include<autd3.hpp>
//~#include <autd3/modulation/fourier.hpp>
//~int main() {
const auto m =
    autd3::modulation::Fourier({autd3::modulation::Sine(100 * autd3::Hz),
                                autd3::modulation::Sine(150 * autd3::Hz)})
        .with_scale_factor(1.f)
        .with_offset(0)
        .with_clamp(true);
//~return 0; }

File: src/codes/Users_Manual/modulation/fourier_3.rs
# use autd3::modulation::Fourier;
# use autd3::prelude::*;
# #[allow(unused_variables)]
# fn main() -> Result<(), Box<dyn std::error::Error>> {
    let m = Fourier::new([Sine::new(100 * Hz), Sine::new(150 * Hz)])?
        .with_scale_factor(Some(1.))
        .with_offset(0)
        .with_clamp(true);
#     Ok(())
# }